### PR TITLE
Minor update, clarification.

### LINKED ISSUE #9077: Clarification
While idempotence mean that same call returns the same result it would be more correct to say that queries are side-effect free in the sentence below:

"The essence of those patterns, and the important point here, is that queries are idempotent: no matter how many times you query a system, the state of that system will not change."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 84d83855-4370-90e8-024a-12c5ac8220e9
* Version Independent ID: 01be3db1-4a00-c8f2-aa59-59656f206b93
* Content: [Applying CQRS and CQS approaches in a DDD microservice in eShopOnContainers](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice#feedback)
* Content Source: [docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md](https://github.com/dotnet/docs/blob/master/docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md)
* Product: **dotnet**
* GitHub Login: @CESARDELATORRE
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by CESARDELATORRE**
Fixes #9077

### CODE REVIEW

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ ms.date: 10/08/2018
 
 The design of the ordering microservice at the eShopOnContainers reference application is based on CQRS principles. However, it uses the simplest approach, which is just separating the queries from the commands and using the same database for both actions.
 
-The essence of those patterns, and the important point here, is that queries are idempotent: no matter how many times you query a system, the state of that system will not change. You could even use a different “reads” data model than the transactional logic “writes” domain model, although the ordering microservices is using the same database. Hence this is a simplified CQRS approach.
+The essence of those patterns, and the important point here, is that queries are idempotent: no matter how many times you query a system, the state of that system will not change. In other words, queries are side-effect free.
```
Comment (mairaw) [inline docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md]:
```suggestion
The essence of those patterns, and the important point here, is that queries are idempotent: no matter how many times you query a system, the state of that system won't change. In other words, queries are side-effect free.
```

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md [hunk 2]
```diff
@@ -9,7 +9,9 @@ ms.date: 10/08/2018
...
+
+Therefore, you could use a different “reads” data model than the transactional logic “writes” domain model, although the ordering microservices is using the same database. Hence this is a simplified CQRS approach.
```
Comment (mairaw) [inline docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md]:
```suggestion
Therefore, you could use a different “reads” data model than the transactional logic “writes” domain model, even though the ordering microservices are using the same database. Hence, this is a simplified CQRS approach.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good to me but left a few suggestions for improving those sentences

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md (hunk 1 above)
Comment (CESARDELATORRE) [reply docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md]: 👍

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md (hunk 2 above)
Comment (CESARDELATORRE) [reply docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/eshoponcontainers-cqrs-ddd-microservice.md]: 👍 Good to go!

**Review by mairaw — APPROVED** [review]
Thanks @CESARDELATORRE!

**Pull request merged**